SeaPublicWebsite/wwwroot/css/_simple-energy-advice-custom.scss

```diff
@@ -269,6 +269,19 @@
     }
 }
 
+.sea-saved-recommendations-table-mobile-hide {
+    @media (max-width: 640px) {
```

Comment:
We now have a few `@media` queries. If you have time, could you please rationalise them along the lines of https://itnext.io/writing-media-queries-with-sass-mixins-3ea591ea3ea4?
If not, then please raise a tech-debt ticket for this.

Reply:
ticket raised